src/Workspaces/Core/Portable/Classification/ClassifiedSpan.cs

```diff
@@ -3,14 +3,18 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Runtime.Serialization;
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.Classification;
 
+[DataContract]
```

Comment:
This is public API, so let me know if this change isn't allowed, and I can create a wrapper in the EA